release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -324,6 +324,11 @@ You can now create a performance profile using the Performance Profile Creator (
 
 For more information, see xref:../scalability_and_performance/cnf-create-performance-profiles.adoc#cnf-create-performance-profiles[Creating a Performance Profile].
 
+[id="ocp-4-8-nfd-operator"]
+==== Node Feature Discovery Operator
+
+The xref:../scalability_and_performance//psap-node-feature-discovery-operator.adoc#node-feature-discovery-operator[Node Feature Discovery (NFD) Operator] is now available. Use it to expose node-level information by orchestrating Node Feature Discovery, a Kubernetes add-on for detecting hardware features and system configuration.
```

Comment:
Link works but does seem to have an extraneous `/`

Reply:
Oh weird! Local build worked great too. Great eye! This is updated :-) Thanks!